modules/mobile/config/routes.rb

```diff
@@ -17,6 +17,7 @@
     post '/appointment', to: 'appointments#create'
     get '/claims-and-appeals-overview', to: 'claims_and_appeals#index'
     get '/claims/decision_letters', to: 'decision_letters#index'
+    get '/claims/decision_letters/:document_id/download', to: 'decision_letters#download'
```

Comment:
I realized I messed up the naming of my endpoint and it got copied over to yours as well. we should be using the kebab `-` instead of the `_`. If you want to change the index too and update the url in the spec then I don't have to merge in my PR to fix it and cause another conflict.

Reply:
Sounds good